coretex/cli/modules/node.py

```diff
@@ -287,10 +307,24 @@ def configureNode(config: Dict[str, Any], verbose: bool) -> None:
 
     if verbose:
         config["storagePath"] = clickPrompt("Storage path (press enter to use default)", config_defaults.DEFAULT_STORAGE_PATH, type = str)
-        config["nodeRam"] = clickPrompt("Node RAM memory limit in GB (press enter to use default)", config_defaults.DEFAULT_RAM_MEMORY, type = int)
+
+        cpuCount = clickPrompt("Enter the number of CPUs the container will use (press enter to use default)", config_defaults.DEFAULT_CPU_COUNT, type = int)
+        if cpuCount > cpuLimit:
+            stdEcho(f"WARNING: CPU limit in Docker Desktop ({cpuLimit}) is lower than the configured value ({config['cpuCount']}). Please adjust resource limitations in Docker Desktop settings.")
+            cpuCount = cpuLimit
+        config["cpuCount"] = cpuCount
+
+        if config_defaults.DEFAULT_RAM_MEMORY < config_defaults.MINIMUM_RAM_MEMORY:
+            errorEcho("Minimum Node RAM requirement is 6GB.")
+            return
+        nodeRam = clickPrompt("Node RAM memory limit in GB (press enter to use default)", config_defaults.DEFAULT_RAM_MEMORY, type = int)
+        if nodeRam > ramLimit:
+            stdEcho(f"WARNING: CPU limit in Docker Desktop ({cpuLimit}) is lower than the configured value ({config['cpuCount']}). Please adjust resource limitations in Docker Desktop settings.")
+            nodeRam = ramLimit
```

Comment:
It should not be allowed to configure Node with lower value compared to limit, this should either prompt user to enter the value again until a valid value is entered.

In RAM case where there is a hard minimum limit it should flat out refuse to configure/start the Node if there is not enough resources.